_utilities/layout-grid.md

```diff
@@ -504,9 +504,9 @@ Add `grid-gap` to a grid row to add a gap (or gutter) between each column in the
 When generating your CSS from USWDS source files, you have the option of customizing many system defaults by modifying project theme variables. USWDS also provides grid mixins for adding grid functionality to custom semantic component CSS.
 
 ### Variables
-Variables and maps determine the number of columns, gutter width, and media-query point at which to begin floating columns. We use variables to generate the predefined grid classes documented on this page, as well as for the custom mixins noted under USWDS-theme-spacing.scss and USWDS-theme-utilities.scss.
+Variables and maps determine the number of columns, gutter width, and media-query point at which to begin floating columns. We use variables to generate the predefined grid classes documented on this page, as well as for the custom mixins noted under Spacing settings and Utilities settings
```

Comment:
Can you link to spacing settings and utilities settings (even if they're in-page)?

Reply:
Sure! They are _directly_ beneath this paragraph but as long as you're cool with that!

Reply:
Resolved in https://github.com/uswds/uswds-site/pull/1866/commits/174fb5284a163aab653d6b1a843d393ae8a56be8